scopes/component/remove/delete-cmd.ts

```diff
@@ -98,12 +98,15 @@ ${chalk.bold('to update the remote, please tag/snap and then export. to revert,
     return removedObjectsArray.map((item) => removeTemplate(item, true));
   }
 
-  private async removePrompt(hard?: boolean) {
+  private async removePrompt(hard?: boolean, lane?: boolean) {
     this.remove.logger.clearStatusLine();
+    const laneOrMainWarning = lane
+      ? `this command will mark the component as removed from this lane, and any changes made on this lane will be reverted after snap and export`
```

Comment:
@benjgil , It's a good idea to have a different message for lanes.
To me though this sentence is not clear:
> and any changes made on this lane will be reverted after snap and export

If you snap and export, the component is still deleted on the lane. Maybe you mean that after merging the lane, this "delete" op won't be relevant?

Reply:
it's not deleted, it's just removed from being part of the lane. i.e. whatever the state of the component on main before this lane wont be changed by the lane. So the actual effect here is that you're reverting any changes to the component made on the lane.

(you would need to add the `--update-main` flag for the component to actually be deleted, and that deletion to propagate to main)

Reply:
though understand how this can be confusing as it is
how about:
`and as a result any changes made to the component on this lane will be reverted`

Reply:
Oh ok I understand. But still the word 'revert' here is confusing to me. I'd say something like 'unaffected on main'

Reply:
@davidfirst coming back to this after a while (having seen this problem again today ;) )

Please see the updated warning - look ok?